.pipeline/config.yml

```diff
@@ -0,0 +1,123 @@
+---
+general:
+  buildTool: "npm"
+  containerRegistryUrl: https://concise.common.repositories.cloud.sap
+  # the chartPath will be activated in general once piper PRs are merged
+#  chartPath: charts/sustainable-saas
+service:
+  buildToolVersion: "N18"
+stages:
+  Build:
+    mavenExecuteStaticCodeChecks: false
+    npmExecuteLint: false
+  Additional Unit Tests:
+    npmExecuteScripts: false
+  Acceptance:
+    cloudFoundryDeploy: false
+    npmExecuteEndToEndTests: false
+    kubernetesDeploy: false
+    # this is where the configuration would go for a kubernetesDeploy during the Acceptance stage
+#    chartPath: charts/sustainable-saas
+#    additionalParameters:
+#      - --debug
+#      - --set
+#      - global.imagePullSecret.name=concise
+#    deployTool: helm3
+#    deploymentName: mh-deployment-01
+#    kubeConfigFileCredentialsId: kube-config
+#    namespace: mh-01
+#    valuesMapping:
+#      api.image.repository: image.susaas_image/susaas_api.repository
+#      api.image.tag: image.susaas_image/susaas_api.tag
+#      srv.image.repository: image.susaas_images/susaas_srv.repository
+#      srv.image.tag: image.susaas_images/susaas_srv.tag
+#      broker.image.repository: image.susaas_image/broker.repository
+#      broker.image.tag: image.susaas_image/broker.tag
+  Compliance:
+    sonarExecuteScan: false
+  Release:
+    cloudFoundryDeploy: false
+    tmsUpload: false
+    kubernetesDeploy: true
+    # the chartPath will be moved to general once piper PRs are merged
+    chartPath: charts/sustainable-saas
+    # here is the configuration for a kubernetesDeploy during the Release stage
+    additionalParameters:
+      # we should keep the debug flag. Otherwise there is not much written to the log during deployment. That look like we are stuck.
+      - --debug
+      - --set
+      - global.imagePullSecret.name=concise
```

Comment:
can I set multiple of these? Where does this come from? Can I set container registry secret somewhere else?